virtualization/Docker/Dockerfile.dev

```diff
@@ -0,0 +1,55 @@
+# Dockerfile for development
+# Based on the production Dockerfile, but with development additions.
+# Keep this file as close as possible to the production Dockerfile, so the environments match.
+
+FROM python:3.5
```

Comment:
Since our minimum required version is 3.4.2 (Raspberry PI), its probably better to keep developing there?